src/design/optForceGAMS/optForceWithGAMS.m

```diff
@@ -0,0 +1,840 @@
+%
+% targetRxn (obligatory):   Type: string
+%                           Description: string containing the ID for the
+%                           reaction whose flux is intented to be increased.
+%                           For example, if the production of succionate is
+%                           desired to be increased, 'EX_suc' should be
+%                           chosen as the target reaction
+%                           Example: targetRxn='EX_suc';
+%
+% mustU (obligatory):       Type: cell array.
+%                           Description: List of reactions in the MustU set
+%                           This input can be obtained by running the
+%                           script findMustU.m
+%                           Alternatively, there is a second usage of this
+%                           input:
+%                           Type: string.
+%                           Description: name of the .xls file containing
+%                           the list of the reactions in the MustU set
+%                           Example first usage: mustU={'R21_f';'R22_f'};
+%                           Example second usage: mustU='MustU';
+%
+% mustL (obligatory):       Type: cell array.
+%                           Description: List of reactions in the MustL set
+%                           This input can be obtained by running the
+%                           script findMustL.m
+%                           Alternatively, there is a second usage of this
+%                           input:
+%                           Type: string.
+%                           Description: name of the .xls file containing
+%                           the list of the reactions in the MustU set
+%                           Example first usage: mustL={'R11_f';'R26_f'};
+%                           Example second usage: mustL='MustL';
+%
+% minFluxesW (obligatory):   Type: double array of size n_rxns x1
+%                            Description: Minimum fluxes for each reaction
+%                            in the model for wild-type strain
+%                            Example: minFluxesW=[-90; -56];
+%
+% maxFluxesW (obligatory):   Type: double array of size n_rxnsx1
+%                            Description: Maximum fluxes for each reaction
+%                            in the model for wild-type strain
+%                            Example: maxFluxesW=[92; -86];
+%
+% minFluxesM (obligatory):   Type: double array of size n_rxnsx1
+%                            Description: Minimum fluxes for each reaction
+%                            in the model for mutant strain
+%                            Example: minFluxesW=[-90; -56];
+%
+% maxFluxesM (obligatory):   Type: double array of size n_rxnsx1
+%                            Description: Maxmum fluxes for each reaction
+%                            in the model for mutant strain
+%                            Example: maxFluxesW=[92; -86];
+%
+% k(optional):              Type: double
```

Comment:
What I would do is the following:
Textual descriptions of output files (small excerpts of file header and similar things) go into the OUTPUT and are provided in the tutorial. The actual file should be generated on the machine of a user and not be distributed via github. The reason is quite simple: if in a tutorial I get all outputs provided and I have no clue about matlab, I might think that something is working while it is actually not working. E.g. if my gams would not be working properly, I can easily imagine users going on with the tutorial if the output file is there and can be further processed without them actually having performed the steps. And later wondering, why it worked in the tutorial but not in their actual code.